engine/installation/index.md

```diff
@@ -53,6 +53,22 @@ platform.
 See also [Docker Cloud](#on-docker-cloud) for setup instructions for
 Digital Ocean, Packet, SoftLink, or Bring Your Own Cloud.
 
+## Time-based release schedule
+
+Starting with Docker 17.03, Docker uses semantic versioning and a time-based
```

Comment:
"Docker uses semantic versioning" is not correct here; we've never used SemVer (our versions only "looked" SemVer), and YY.MM is definitely not SemVer (https://semver.org)